chromium_src/chrome/common/importer/edge_importer_utils_win.cc

```diff
@@ -0,0 +1,40 @@
+/* Copyright (c) 2022 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#include <Shlobj.h>
+
+#include "base/files/file.h"
+#include "base/files/file_path.h"
+#include "base/files/file_util.h"
+#include "base/logging.h"
+#include "base/win/windows_version.h"
+
+namespace importer {
+
+base::FilePath GetEdgeDataFilePath() {
+  wchar_t buffer[MAX_PATH];
+  if (::SHGetFolderPath(NULL, CSIDL_LOCAL_APPDATA, NULL, SHGFP_TYPE_CURRENT,
+                        buffer) != S_OK)
+    return base::FilePath();
+
+  base::FilePath base_path(buffer);
+  return base_path.Append(L"Microsoft\\Edge\\User Data\\Default");
```

Comment:
So this is hardcoded to the default profile; which I think is OK to start (maybe we can create a follow up issue for multiple profiles).

If you see the existing experience for Chromium, it's showing multiple profiles:
![image](https://user-images.githubusercontent.com/4733304/176394163-74ad7090-4886-4f07-8f61-5dd6b1868b75.png)

We have another problem (not related to this pull request) where it's hard to tell which profile belongs to which browser 😂 In my case, I have Edge with multiple profiles, Chrome with multiple profiles, Chrome canary, and then Chromium (and Firefox old version and newer FF version; they have two profile formats I think)

Reply:
<img width="343" alt="image" src="https://user-images.githubusercontent.com/2965009/176428812-914d6b97-7c1f-47af-9722-49fde54ec2c6.png">
fixed